### PR TITLE
Fixed typos with $SSLCERTDOMAIN & $SSLCERTHOST vars when generating snake oil certs.

### PR DESCRIPTION
**Pull request by slmingol**

### CODE REVIEW

**Comment by rroemhild** [comment]
oh :) thank you for the pr!

**Comment by slmingol** [comment]
No problem. Was trying to figure out how to paramaterize `$XMPP_DOMAIN` with a list of virtual domains as well as passing in a list of SSL certs in. I'd opted to use `-v` and add multiple XMPP_DOMAIN1-4 to your setup but would've much rather figured out a way to loop through a list of them on `$XMPP_DOMAIN` instead.

````
run --name=ejabberd -d -e XMPP_DOMAIN1=somedom1.com -e XMPP_DOMAIN2=somedom2.com \
	-e XMPP_DOMAIN3=somedom3.com -e XMPP_DOMAIN4=somedom4.com \
	-v /var/docker/ejabberd/ssl:/opt/ejabberd/ssl \
        -v /var/docker/ejabberd/conf/ejabberd.yml:/opt/ejabberd/conf/ejabberd.yml \
	-v /var/docker/ejabberd/conf/ejabberdctl.cfg:/opt/ejabberd/conf/ejabberdctl.cfg \
....
````

Another mod that I added but didn't PR it since it isn't fully working yet was a check if exists on the CONFIGFILES that you regen from the .tpl files. But I believe they already do since they're part of the ejabberd download. 

Basically I didn't want to have to buld my own image, preferring to "override" yours via configs. Just some additional thoughts, thanks again for a solid image though!

**Comment by rroemhild** [comment]
I have some ideas to add multible xmpp domains with the env var. To track this I opened a new issue.

> Another mod that I added but didn't PR it since it isn't fully working yet was a check if exists on the CONFIGFILES that you regen from the .tpl files. But I believe they already do since they're part of the ejabberd download.

The Ejabber installer setup basic config files to the install dir:

```sh
$ docker run -d -P --name ejabberd rroemhild/ejabberd
$ docker exec ejabberd ls -l /opt/ejabberd/conf
```
